<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiFunctionalDashBoard</class>
 <widget class="QMainWindow" name="multiFunctionalDashBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>multiFunctionalDashBoard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="hourLCD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>11.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteLCD">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>11.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secondLCD">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>11.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="imageDownloadDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Download/Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>ImageLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>113</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Download/Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>ImageLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherIcon">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Weather Icon</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
